docs/tutorials/configure/scuttlebot.md

```diff
@@ -159,164 +240,185 @@ This will be similar to adding the right motor.
 
-    The `left` and `right` attributes represent the motors corresponding to the left and right sides of the rover.
-    Since we named the motors "left" and "right", you can simply add "left" and "right" between the brackets for your set of motors, respectively.
+{{< tabs name="Configure a Wheeled Base" >}}
+{{% tab name="Config Builder" %}}
 
-The attributes of your base component's config should look something like this:
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your base, select the type `base`, and select the `wheeled` model.
 
-```json {class="line-numbers linkable-line-numbers"}
+Click **Create component**.
+
+{{< imgproc src="/components/base/wheeled-base-ui-config.png" alt="An example configuration for a wheeled base." resize="600x" >}}
+
+1. Select `right` as the **Right Motor** and `left` as the **Left Motor**.
+2. Enter "400" in `width_mm` (measured between the midpoints of the wheels).
+3. Enter "250" in `wheel_circumference_mm`.
+
+{{% /tab %}}
+{{% tab name="Raw JSON" %}}
+
+```json
 {
- "width_mm": 400,
- "wheel_circumference_mm": 250,
- "left": ["left"],
- "right": ["right"]
+  "components": [
+    {
+      "attributes": {},
+      "model": "pi",
+      "name": "local",
+      "type": "board"
+    },
+    {
+      "attributes": {
+        "board": "local",
+        "pins": {
+          "dir": "16",
+          "pwm": "15"
+        }
+      },
+      "model": "gpio",
+      "name": "right",
+      "type": "motor"
+    },
+    {
+      "attributes": {
+        "board": "local",
+        "pins": {
+          "dir": "13",
+          "pwm": "11"
+        }
+      },
+      "model": "gpio",
+      "name": "left",
+      "type": "motor"
+    },
+    {
+      "attributes": {
+        "left": [ "left" ],
+        "right": [ "right" ],
+        "wheel_circumference_mm": 250,
+        "width_mm": 400
+      },
+      "model": "wheeled",
+      "name": "scuttle-base",
+      "type": "base"
+    }
+  ]
 }
 ```
 
-When you save the config and switch to the **Control** tab, you'll see new control buttons for the base.
-In the **Keyboard** tab, toggle your keyboard control to active.
-Use **W** and **S** to go forward and back, and **A** and **D** to arc and spin.
+{{% /tab %}}
+{{< /tabs >}}
 
-{{<imgproc src="/tutorials/scuttlebot/pi-kybrd-control.png" resize="300x" declaredimensions=true alt="WASD controls">}}
+Save the config by clicking **Save config** at the bottom of the page.
 
-If you click the **Discrete** tab, then you'll see different movement modes such as `Straight` and `Spin`; and different movement types such as `Continuous` and `Discrete` and directions such as `Forwards` and `Backwards`.
+### Test the base
 
-![Discrete controls](/tutorials/scuttlebot/pi-discrete.png)
+Now that you have configured the base you can try moving the SCUTTLE Robot with your keyboard.
```

Comment:
```suggestion
Now that you have configured the base you can try moving the SCUTTLE robot with your keyboard.
```
[opt] I think this isn't part of the product name? It's a bit inconsistent in https://www.scuttlerobot.org/resource/guide/beginners-guide/ but it seems that it's just called a "SCUTTLE" in most places in their instructions etc.

Reply:
we have a rule to capitalize it like that based on style guide - I do see it capitalized like that once and one occurance of it not capitalized

Reply:
was introduced by Charlotte

Reply:
Yeah, that piece of the style guide came about because we had a lot of instances of "Scuttle" instead of "SCUTTLE." I marked this as opt because I also saw in the thing I linked that there is one of each, but if you look into the embedded PDF, there are numerous instances of "SCUTTLE" as a complete noun with no "robot" or "Robot" after it. So really it seems they would be most likely to say
```suggestion
Now that you have configured the base you can try moving the SCUTTLE with your keyboard.
```

Reply:
ok all changed